cron/maintenance/serverless.yml

```diff
@@ -35,4 +35,4 @@ functions:
   cron:
     handler: bin/maintenance
     events:
-      - schedule: cron(30 * * * ? *)
+      - schedule: cron(0 4 * * SUN *)
```

Comment:
Sorry, I should have checked this - it looks like you need a `?` for the day of the month.
```One of the day-of-month or day-of-week values must be a question mark (?).```

Reply:
https://docs.aws.amazon.com/lambda/latest/dg/tutorial-scheduled-events-schedule-expressions.html

Reply:
Thanks for finding the problem. I remember reading that but somehow convinced myself it wasn't needed here.